### PR TITLE
Renamed blocked_position, created the functions: game_processing, inp…

### PR DESCRIPTION
**Pull request by epsoon**
Renamed blocked_position, created the functions: game_processing, input_processing_start, input_processing_game and point_processing.

### CODE REVIEW

File: pahtum.py [hunk 1]
```diff
@@ -27,22 +27,84 @@ def block_fields(board):
...
-    return board
-    
+    return board, blocked_positions
+
+def input_processing_game(blocked_positions):
+    upper_row = ['A', 'B', 'C', 'D', 'E', 'F', 'G']
+    left_column = ['1', '2', '3', '4', '5', '6', '7']
+    while True:
+        user_input = input('Please enter a valid command (2A, 5B, etc..): ') # Phrase is just a placeholder...
+        if len(user_input) >= 2 and user_input[0] in left_column and user_input[1] in upper_row:
```
Comment (ElviraMingazova) [inline pahtum.py]: I think len(user_input) should be equal to 2 (len(user_input) == 2) because if it's longer is not correct.

Comment (epsoon) [reply pahtum.py]: Does not matter in this case bcs it also checks whether the input is in left_column and upper_row. I just wanted to specifiy that inputs like '' or '1' are not allowed.  You can change the line anyway.

File: pahtum.py [hunk 2]
```diff
@@ -27,22 +27,84 @@ def block_fields(board):
...
+            number_index = int(user_input[0])
+            letter_index = int(upper_row.index(user_input[1]) + 1)
```
Comment (ElviraMingazova) [inline pahtum.py]: +1 is not needed, I just checked. The letter is already at the right index.

File: pahtum.py (hunk 1 above)
Comment (ElviraMingazova) [reply pahtum.py]: You are right, the program would not crash, but an input like '5Gwhatever' would be allowed and would not cause any trouble but wouldn't be very nice to accept.

File: pahtum.py (hunk 2 above)
Comment (epsoon) [reply pahtum.py]: I think the +1 is needed. You should confirm this while running the program.

**Pull request merged**